src/main/java/org/fcrepo/camel/RdfNamespaces.java

```diff
@@ -29,6 +33,20 @@
 
     public static final String LDP = "http://www.w3.org/ns/ldp#";
 
+    public static final String SERVER_MANAGED = REPOSITORY + "ServerManaged";
```

Comment:
Good point. It's possible that these `String` values would be used in running code, but I doubt that. The `Map<String, String>` is used in `String FcrepoProducer.addPreferNamespace(String)`.